helpdesk_mgmt_ticket_tree_decoration/views/helpdesk_ticket_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="ticket_view_tree" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.tree</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="priority" eval="100"/>
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="." position="attributes">
+                <attribute name="decoration-danger">unattended</attribute>
+                <attribute name='decoration-success'>closed</attribute>
+            </xpath>
+            <field name="stage_id" position="after">
+                <field name="unattended" invisible="True"/>
+                <field name="closed" invisible="True"/>
+            </field>
```

Comment:
Suggested improvement (archived tickets will be shown grayed):
```suggestion
            <xpath expr="." position="attributes">
                <attribute name="decoration-muted">not active</attribute>
                <attribute name="decoration-danger">active and unattended</attribute>
                <attribute name='decoration-success'>active and closed</attribute>
            </xpath>
            <field name="stage_id" position="after">
                <field name="active" invisible="True"/>
                <field name="unattended" invisible="True"/>
                <field name="closed" invisible="True"/>
            </field>
```

Reply:
Done! 
Although as active and non-active are never mixed, when the non-active ones are shown they will all be "muted"...